membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -155,33 +155,41 @@ def test_core_fast_leaflets():
         assert_almost_equal(z, z_test)
 
 
-@pytest.mark.parametrize('dummy_coordinates, test_mapper, n_cells, max_width', [(
-    # dummy coordinates (x,y)
-    ((0, 0), (1, 0), (2, 0),
-     (0, 1), (1, 1), (2, 1),
-     (0, 2), (1, 2), (2, 2)),
-    # should map to
-    lambda xy: (xy[0], xy[1]),
-    3, 3),
-])
-def test_grid_map_small_9grid(dummy_coordinates, test_mapper, n_cells, max_width):
-    factor = np.float32(n_cells / max_width)
-    for dummy_coord in dummy_coordinates:
-        assert test_mapper(dummy_coord) == grid_map(dummy_coord, factor)
-
-
-@pytest.mark.parametrize('dummy_coordinates, test_mapper, n_cells, max_width', [(
-    # dummy coordinates (x,y)
+@pytest.mark.parametrize('grid', [(
```

Comment:
```suggestion
@pytest.mark.parametrize('factor', [1, 2])
@pytest.mark.parametrize('grid', [(
```

One fun thing about `parametrize` is that you can next them. For example, adding this `factor` line will result in four cases:

* factor = 1, grid = first grid
* factor = 1, grid = second grid
* factor = 2, grid = first grid
* factor = 2, grid = second grid

This can let you reduce the test itself a little, because now each iteration only tests 1 factor value and 1 grid value.